### ISSUE TITLE
Feature Request: Show available space on server and connection details in bottom status line

### ISSUE THREAD

**Issue by emk2203** [equeim/tremotesf2#374]
Comparing with TransGUI, the features I am missing most are

1. The available space on the server in the bottom status line
2. Daemon details like `Transmission 4.0.4 (24077e3511) in 192.168.88.39:9091` with daemon version and IP address:port instead of just the name, which is redundant and can be gotten from the `tremotesf2` settings.
3. Optional Three-pane view instead of double-clicking torrents

If you connect to several servers, 1. helps you to see if a server is running out of space before adding torrents. 2. is helpful for debugging if Transmission versions are not whitelisted by some trackers. 3. is also helpful for debugging since you can jump from torrent to torrent quickly and compare them if some have issues.

**Comment by bassstorm** [comment]
I double that request :-)

**Comment by Revenger** [comment]
Agreed, I have a server with multiple drives mounted that I download to would be nice to show the space on each drive.
disk 1 - 50gb free
disk 2 - 100gb free
etc

**Comment by Pentaphon** [comment]
>in the bottom status line

Yes! It should always be on the main window.

**Comment by antivirtel** [comment]
Dupe of #246

**Comment by Pentaphon** [comment]
> Dupe of #246 ?

Yep. @equeim
